sklearn/cross_decomposition/_pls.py

```diff
@@ -964,45 +979,52 @@ def __init__(self, n_components=2, *, scale=True, copy=True):
         self.copy = copy
 
     @_fit_context(prefer_skip_nested_validation=True)
-    def fit(self, X, Y):
+    def fit(self, X, y, Y=None):
```

Comment:
Also here.

Reply:
Thanks for all the feedback I will get to work on these points.

Reply:
@glemaitre One quick question. Should I rename Y to y in all functions in the module, or just in the previously worked fit functions?

Reply:
I did not look at the rest of the module but I think this is reasonable to do it for the full `_pls.py` module.